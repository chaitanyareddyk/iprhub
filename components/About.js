import React from 'react';

const About = () => {
  return (
    <div>
      <b>About us</b>
    </div>
  );
};

export default About;
