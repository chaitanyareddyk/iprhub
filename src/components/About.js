import React, { Component } from "react";
import { Grid, Header } from "semantic-ui-react";
import "semantic-ui-css/semantic.min.css";

export default class About extends Component {
  render() {
    return (
      <div style={{ minHeight: 700, padding: "8em 0em" }}>
        <Grid stackable container>
          <Grid.Row>
            <Grid.Column>
              <Header as="h1" textAlign="center" style={{ fontSize: "2.3em" }}>
                About Us
              </Header>
              <p textAlign="center" style={{ fontSize: "1.33em" }}>
                We are a group of people, who are working 100% to bring lawyers
                and inventors on one platform to make their work easier for both
                the parties, while minimizing the costs due to third parties...
              </p>
            </Grid.Column>
          </Grid.Row>
        </Grid>
      </div>
    );
  }
}
