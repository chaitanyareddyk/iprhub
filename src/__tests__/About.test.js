import renderer from "react-test-renderer";
import About from "../components/About";
import React from "react";

test("About component renders the about us correctly", () => {
  const rendered = renderer.create(<About />);
  expect(rendered.toJSON()).toMatchSnapshot();
});
