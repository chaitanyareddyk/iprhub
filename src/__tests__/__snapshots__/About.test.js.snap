// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders the about us correctly 1`] = `
<div
  style={
    Object {
      "minHeight": 700,
      "padding": "8em 0em",
    }
  }
>
  <div
    className="ui container stackable grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      >
        <h2
          className="ui left aligned header"
        >
          About Us
        </h2>
        <div
          className="ui divider"
        />
        <p
          style={
            Object {
              "fontSize": "1.33em",
            }
          }
        >
          We are a group of people, who are working 100% to bring lawyers and inventors on one platform to make their work easier for both the parties, while minimizing the costs due to third parties...
        </p>
      </div>
    </div>
  </div>
</div>
`;
